src/rpc/blockchain.cpp

```diff
@@ -996,14 +998,29 @@ static UniValue gettxoutsetinfo(const JSONRPCRequest& request)
     CCoinsStats stats;
     ::ChainstateActive().ForceFlushStateToDisk();
 
+    CoinStatsHashType hash_type = CoinStatsHashType::HASH_SERIALIZED;
```

Comment:
nit, move code below to new function and here make it `ParseHashType(request.params[0], CoinStatsHashType::HASH_SERIALIZED)` (2nd is default value if 1st is null).

Reply:
done